chaos.py

```diff
@@ -1,34 +1,22 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import time
 import os
+from os.path import dirname, abspath, join
+
+import time
 import sys
 import logging
-import threading
-import http.server
-import random
 import subprocess
 import settings
-import patch
 import schedule
 
-from os.path import dirname, abspath, join
-
 import cron
 import github_api as gh
-import github_api.prs
-import github_api.voting
-import github_api.repos
```

Comment:
This is [used on master](https://github.com/chaosbot/Chaos/blob/c2498c4621a5f890bd704d568a9192acd9ab5711/chaos.py#L53).